README.md

```diff
@@ -66,6 +66,14 @@ This project is licensed under [the MIT License](LICENSE)
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+## Interactions
+
+You can report bugs or submit feature requests in the [repository's Issues section](https://github.com/Azure/azure-api-management-devops-resource-kit/issues).
+
+Azure API Management DevOps resourceKit has limited "assited support". You can also use [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-api-management-devops-resource-kit) with the `azure-api-management-devops-resource-kit` tag.
```

Comment:
Some nits:
```suggestion
Azure API Management DevOps resource kit has limited "assited support". You can use [GitHub Discussions](https://github.com/Azure/azure-api-management-devops-resource-kit/discussions/new) or [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-api-management-devops-resource-kit) with the `azure-api-management-devops-resource-kit` tag.
```

I think it's good to keep the context as close to the repo as we can.

You might want to rephrase "assited support" - What does that mean? Can customers still open support tickets?